enhancements/storage/storage-hypershift.md

```diff
@@ -0,0 +1,973 @@
+| assets/aws-ebs/02_sa.yaml                                             | N/A / CSO (not in MVP) | MC        | ServiceAccount of AWSO                                                                                                                                              |
+| assets/aws-ebs/03_role.yaml                                           | N/A? (not in MVP)      | N/A?      | AWSO’s RBAC in the AWSO namespace (i.e. not much)                                                                                                                   |
+| assets/aws-ebs/04_rolebinding.yaml                                    | N/A? (not in MVP)      | N/A       | AWSO’s RBAC in the AWSO namespace (i.e. not much)                                                                                                                   |
+| assets/aws-ebs/05_clusterrole.yaml                                    | CSO (not in MVP)       | G         | AWSO’s RBAC in the guest cluster (lot of perms.)                                                                                                                    |
+| assets/aws-ebs/06_clusterrolebinding.yaml                             | CSO (not in MVP)       | G         | AWSO’s RBAC in the guest cluster (lot of perms.)                                                                                                                    |
+| assets/aws-ebs/07_role_aws_config.yaml                                | CSO (not in MVP)       | G         | RBAC to allow AWSO to read ConfigMaps in openshift-config-managed namespace in the guest cluster                                                                    |
+| assets/aws-ebs/08_rolebinding_aws_config.yaml                         | CSO (not in MVP)       | G         | RBAC to allow AWSO to read ConfigMaps in openshift-config-managed namespace in the guest cluster                                                                    |
+| assets/aws-ebs/09_deployment.yaml                                     | CSO                    | MC        | AWSO Deployment                                                                                                                                                     |
+| assets/aws-ebs/10_cr.yaml                                             | CSO                    | G         | AWSO CR                                                                                                                                                             |
+| aws-ebs-csi-driver-operator/assets/cabundle_cm.yaml                   | AWSO?                  | G         | ConfigMap where ?? will inject cluster CA bundle                                                                                                                    |
```

Comment:
The service-ca operator should do the right thing in the guest cluster.

Reply:
Updated